storm-client/src/jvm/org/apache/storm/executor/ExecutorShutdown.java

```diff
@@ -60,16 +60,7 @@ public ExecutorStats renderStats() {
 
     @Override
     public void credentialsChanged(Credentials credentials) {
-        TupleImpl tuple = new TupleImpl(executor.getWorkerTopologyContext(), new Values(credentials),
-                                        Constants.SYSTEM_COMPONENT_ID, (int) Constants.SYSTEM_TASK_ID,
-                                        Constants.CREDENTIALS_CHANGED_STREAM_ID);
-        AddressedTuple addressedTuple = new AddressedTuple(AddressedTuple.BROADCAST_DEST, tuple);
-        try {
-            executor.getReceiveQueue().publish(addressedTuple);
-            executor.getReceiveQueue().flush();
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
+        executor.needToRefreshCreds = true;
```

Comment:
Minor: Does this method name and signature needs to change since it is setting a flag and not using the supplied parameter?

Reply:
Since the _SpoutExecutor_ and _BoltExecutor_ during init uses the initialCredentials,  the _accept_ method does not need to worry about it.

Reply:
e.g. https://github.com/apache/storm/blob/master/storm-client/src/jvm/org/apache/storm/executor/spout/SpoutExecutor.java#L144-L146